src/gui/dialogs/window_dlg_strong_warning.cpp

```diff
@@ -12,6 +13,7 @@ window_dlg_strong_warning_t::window_dlg_strong_warning_t()
 
 void window_dlg_strong_warning_t::Show(string_view_utf8 txt) {
     static window_dlg_strong_warning_t dlg;
+    Sound_Play(eSOUND_TYPE::StandardAlert);
```

Comment:
Are you sure this is what you want? To always play an alert sound for every warning dialog?

Reply:
Play StandardAlert only for fan error.